### PR TITLE
Add vonic@1.1.2 w/ npm auto-update

### PR DESCRIPTION
**Pull request by clairetsai818**
Pull request for issue: #10649 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/wangdahoo/vonic/
 * Official website (optional, not the repository): https://wangdahoo.github.io/vonic/docs/
 * NPM package url (optional): https://www.npmjs.com/package/vonic
 * License and its reference: [MIT](https://github.com/wangdahoo/vonic/issues/75)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers:82
   - Count of stars:1546
   - Count of forks:215
 * NPM download stats (optional):
   - Downloads in the last day:8
   - Downloads in the last week:370
   - Downloads in the last month:5199

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by clairetsai818** [comment]
This PR is waiting for confirming the license info with the author.

**Comment by clairetsai818** [comment]
@cdnjs/intern3 Please help me review this PR, thanks!

File: ajax/libs/vonic/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "ui components",
+    "spa"
+  ],
+  "author": "wangdahoo <157195705@qq.com>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/wangdahoo/vonic/"
+  },
+  "license": "MIT",
+  "homepage": "https://wangdahoo.github.io/vonic-documents",
```
Comment (hare1039) [inline ajax/libs/vonic/package.json]: I think docs in not homepage. The one you provide in the PR description is better.
`https://wangdahoo.github.io/vonic/docs/`
What do you think?

Comment (clairetsai818) [inline ajax/libs/vonic/package.json]: @hare1039 thanks! I think this one is better.

**Comment by clairetsai818** [comment]
@cdnjs/intern3 I found the old versions which can't be fetched by auto-update config don't have `vonic.js` file. Do you think that it's okay to include only `src/index.js`?

**Comment by extend1994** [comment]
@clairetsai818 I think they are totally different file, and you can ask the author about which files we need :smiley:

**Comment by clairetsai818** [comment]
@cdnjs/intern3 The author said we only need to host v1.0+, so I only rebase this PR without adding the  versions manually.
ref: https://github.com/wangdahoo/vonic/issues/93

File: ajax/libs/vonic/package.json (hunk 1 above)
Comment (extend1994) [inline ajax/libs/vonic/package.json]: Please modify it as we discussed. (https://wangdahoo.github.io/vonic/docs/)

Comment (clairetsai818) [inline ajax/libs/vonic/package.json]: @extend1994 Sorry that I missed that, and I've edited `homepage` info, please help me review this!

**Pull request merged**